pages/advocates/index.vue

```diff
@@ -79,10 +79,25 @@ main {
   justify-content: space-around;
 }
 
-@media (max-width: 720px) {
+.card {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  background-color: white;
+  padding: 1em;
+  box-shadow: 10px 10px 11px -10px var(--gray-shadow);
+  border: 1px solid var(--gray-shadow);
+  width: 25%;
```

Comment:
I think I explained myself badly. I was not saying to move everything to the page, I was talking about boundary and external properties from a "box point of view". For instance, `display: flex` decides how its inner elements fit inside; the same with `flex-direction`,  `text-align`, `background-color`, `padding`...

But `box-shadow`, `width`, `border` (debatable, it's up to you) and `margin` are aspects of how the page displays the components. Perhaps the page is a flat design with no shadows, perhaps it is a full-width design making its elements to go from the beginning to the end of the viewport, that's the idea.

Anyway, for making it simple, my last comment talked only about the `width` property, so it is not needed you think about each of the properties mentioned before, just restore all of this to the original position but leave the `width` here.